samples/mpsl/timeslot/src/main.c

```diff
@@ -29,9 +29,17 @@ LOG_MODULE_REGISTER(main, LOG_LEVEL_INF);
 
 #if defined(CONFIG_SOC_SERIES_NRF53X)
 	#define LOG_OFFLOAD_IRQn SWI1_IRQn
+	#define MPSL_TIMER0 NRF_TIMER0
 #elif defined(CONFIG_SOC_SERIES_NRF52X)
 	#define LOG_OFFLOAD_IRQn SWI1_EGU1_IRQn
-#endif
+	#define MPSL_TIMER0 NRF_TIMER0
```

Comment:
We should add the `MPSL_TIMER0` define to `mpsl_timeslot.h` so that this can be reused by others

Reply:
Discussed offline. Change will be done after release.